src/Compilers/CSharp/Test/Semantic/Semantics/SwitchTests.cs

```diff
@@ -2949,6 +2949,42 @@ public static void Main()
                 Diagnostic(ErrorCode.WRN_UnreachableCode, "break").WithLocation(14, 17));
         }
 
+        [Fact, WorkItem(47878, "https://github.com/dotnet/roslyn/issues/47878")]
+        public void Bug47878()
+        {
+            var text = @"
+using System;
+public class C
+{   
+    public static void Main()
+    {
+        int x1 = (1, 1, 1, 1, 1, 1, 1, 1, 1, 1) switch
+        {
+            (1, 1, 1, 1, 1, 1, 1, 1, 1, 1) => 1,
```

Comment:
The `11` doesn't seem to line up with every 7-th position.

Reply:
@jcouv I didn't get this.

Reply:
Seems like there was a display bug in the code review tool I use. It displayed `(1, 1, 1, 1, 1, 1, 1, 11, 1, 1, 1, 1, 1, 11, ...)`.
Sorry about that!